.github/workflows/continuous-integration-unit-php.yml

```diff
@@ -105,6 +99,15 @@ jobs:
       - name: Set up tests
         run: bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1:${{ job.services.mysql.ports['3306'] }} ${{ matrix.wp }} true
 
+      # 1. Installs PHPUnit 7.5 and patches it to run on PHP 8.
```

Comment:
As we have to install a different version now, could invert this. Install 7.5 by default and install 5.7 on PHP 5.6?

Reply:
Because of the platform requirement in `composer.json` (PHP 5.6), we can't install 7.5 by default as it only supports PHP 7.1+. So we would need to keep the current default.

Theoretically, on CI we could install PHPUnit 6 on PHP 7.0, and PHPUnit 7.5 for 7.1+.

I don't think installing PHPUnit as a PHAR like some other deps (phpstan, etc.) works, as we then cannot patch the composer autoloader.

Reply:
We could just add a param to the matrix like this 

```
        php: ['7.0', '5.6']
        wp: ['latest']
        coverage: [false]
        phpunit: ['5.7']
        include:
```
```
       - php: '8.0'
          wp: 'latest'
          phpunit: ['7.5']
```

Reply:
I don't see a huge benefit at the moment, except for a small performance boost, but perhaps you could give it a try in an ew PR?

Reply:
Just an idea. I will follow up in a new PR.